io.gitlab.gwendalj.package-transporter.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id" : "io.gitlab.gwendalj.package-transporter",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "package-transporter",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=/var/lib/flatpak/app:ro",
+        "--filesystem=~/.var/app/:ro"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+          "name": "blueprint-compiler",
```

Comment:
there should be a cleanup rule to remove it all since it's only a build time dependency

Reply:
https://github.com/flathub/io.posidon.Paper/blob/79c31579c9e0ae987357410864aa6ed6372d9bad/io.posidon.Paper.json#L42